BackendBench/scripts/main.py

```diff
@@ -98,7 +97,7 @@ def setup_logging(log_level):
 )
 @click.option(
     "--torchbench-data-path",
-    default=DEFAULT_HUGGINGFACE_URL,
+    default=None,
```

Comment:
is the idea never to have a default now? In which case maybe should update README.md in some way to tell people what to put in

Reply:
Yeah I'll clarify it in the readme. Effectively, if left none it'll pull it from huggingface.

Reply:
So there is a default value haha, just add it, easier than updating README

Reply:
I updated the flag to indicate folks shouldn't touch it. Having access to local datasets ends up still being useful as we are updating things.